src/PolyfillTrait6.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/*
+ * This file is part of PHPUnit Good Practices.
+ *
+ * (c) Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PHPUnitGoodPractices\Polyfill;
+
+/**
+ * @internal
+ */
+trait PolyfillTrait6
+{
+    public function expectException($exception)
```

Comment:
@sanmai @keradus can't these methods be `protected`?

Adding `|expect` in [here](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5269/files) would not be needed if they can.

Reply:
I don't think we can: https://3v4l.org/ZQUDv

Reply:
Yeah, original `expectException` method is public, I've assumed it's protected.